changes/tasks.py

```diff
@@ -548,8 +554,54 @@ def switch_channel(self, change):
                     "SMS means you will not receive any messages. You can stop "
                     "your MomConnect messages completely by replying 'STOP'"
                 )
-            reason = change.data and change.data.get("reason")
-            if reason != "whatsapp_unsent_event":
+                fail_contact_check_text = translation.ugettext(
+                    "It seems you dont have an active Whatsapp account. "
+                    "MomConnect msgs for kids aged 1-2 are only on WA. "
+                    "To stop msgs, reply 'STOP' (std rates apply)"
+                )
+                unsent_event_text = translation.ugettext(
+                    "Sorry - we can't send WhatsApp msgs to this phone. "
+                    "MomConnect msgs for kids aged 1-2 are only on WA. "
+                    "To stop msgs, reply 'STOP' (std rates apply)"
+                )
+
+            if reason == "whatsapp_contact_check_fail":
+                utils.ms_client.create_outbound(
+                    {
+                        "to_identity": change.registrant_id,
+                        "content": fail_contact_check_text,
+                        "channel": "sms",
+                        "metadata": {
+                            "template": {
+                                "name": "mc_important_info",
+                                "language": utils.WHATSAPP_LANGUAGE_MAP[
+                                    subscriptions[0]["lang"]
+                                ],
+                                "variables": [fail_contact_check_text],
+                            }
+                        },
+                    }
+                )
+
+            elif reason == "postbirth_wa_unsent_event":
+                utils.ms_client.create_outbound(
+                    {
+                        "to_identity": change.registrant_id,
+                        "content": unsent_event_text,
+                        "channel": "sms",
```

Comment:
I think that the channel name here is `JUNE_TEXT`.

Also, we won't need the template metadata for SMS.

Reply:
ahh I see, thank you